kyo-scheduler/jvm/src/main/scala/kyo/scheduler/util/Top.scala

```diff
@@ -94,52 +95,54 @@ object Top extends App {
         val sb = new StringBuilder()
 
         sb.append(f"""
-            |===============================================================================================
-            |Kyo Scheduler Status      | LoadAvg: ${status.loadAvg}%1.4f   | Flushes: ${status.flushes}
-            |===============================================================================================
-            |Regulator   |   %% | Allow | Reject | Probes | Cmpl  | Adjmts | Updts |    Avg    |  Jitter
-            |-----------------------------------------------------------------------------------------------
+            |╔═══════════════════════════════════════════════════════════════════════════════════════════════════╗
+            |║ *..*..*.   *.    .. *    *  *  .*.  Kyo Scheduler Top  .  . *   .   *  . * *    .*.   .*.   ...*. ║ 
+            |╚═══════════════════════════════════════════════════════════════════════════════════════════════════╝
```

Comment:
All that's left are some pretty colors for increases/decreases in utilization.

Reply:
that'd be cool! :)